utils/kubernetes/kompose/convert.go

```diff
@@ -57,77 +46,28 @@ func Convert(dockerCompose DockerComposeFile) (string, error) {
 		return "", ErrCvrtKompose(err)
 	}
 
-	ConvertOpt := kobject.ConvertOptions{
-		ToStdout:     false,
-		CreateChart:  false, // for helm charts
-		GenerateYaml: true,
-		GenerateJSON: false,
-		Replicas:     1,
-		InputFiles:   []string{"temp.data"},
-		OutFile:      "result.yaml",
-		Provider:     "kubernetes",
-		CreateD:      false,
-		CreateDS:     false, CreateRC: false,
-		Build:                       "none",
-		BuildRepo:                   "",
-		BuildBranch:                 "",
-		PushImage:                   false,
-		PushImageRegistry:           "",
-		CreateDeploymentConfig:      true,
-		EmptyVols:                   false,
-		Volumes:                     "persistentVolumeClaim",
-		PVCRequestSize:              "",
-		InsecureRepository:          false,
-		IsDeploymentFlag:            false,
-		IsDaemonSetFlag:             false,
-		IsReplicationControllerFlag: false,
-		Controller:                  "",
-		IsReplicaSetFlag:            false,
-		IsDeploymentConfigFlag:      false,
-		YAMLIndent:                  2,
-		WithKomposeAnnotation:       true,
-		MultipleContainerMode:       false,
-		ServiceGroupMode:            "",
-		ServiceGroupName:            "",
+	komposeClient, err := client.NewClient()
+	if err != nil {
+		return "", ErrCvrtKompose(err)
 	}
 
-	err = convert(ConvertOpt)
-	if err != nil {
-		return "", err
+	ConvertOpt := client.ConvertOptions{
```

Comment:
You might want to add the `WithKomposeAnnotations` option, as it was previously being passed.  
However, I’m not sure if it is being put to any use.